Src/Web/Web.Shared.Net.Tests/RequestTrackingTelemetryModuleTest.cs

```diff
@@ -495,10 +521,10 @@ private string GetActivityRootId(string telemetryId)
             return telemetryId.Substring(1, telemetryId.IndexOf('.') - 1);
         }
 
-        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null)
+        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null, CorrelationIdLookupHelper correlationHelper = null)
```

Comment:
For a service of serious scale - do you really think it matters if the first few calls are missing the x-component correlation? 
If would say - let someone ask for it before we give it to them.